packages/react-app/src/components/AddressInput.jsx

```diff
@@ -1,29 +1,25 @@
 import { CameraOutlined, QrcodeOutlined } from "@ant-design/icons";
-import { Badge, Input } from "antd";
-import { useLookupAddress } from "eth-hooks/dapps/ens";
+import { Badge, Input, message, Spin } from "antd";
+import { useLookupAddress } from "eth-hooks";
```

Comment:
I was just starting the ethereumSpeedRun and found out that this change was braking my fresh build D:

Reply:
sameee

Reply:
My bad let me fix that

Reply:
Just added a temporary fix but as soon as eth-hooks will be updated the addressInput.jsx will change